lib/parsers/asm-parser-tic2000.js

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) 2019, Ethan Slattery
```

Comment:
Please, update this. Very importantly, unless you are Ethan Slattery, either put your own name or "Compiler Explorer Authors".

And please make it a typescript file too. Thanks!